src/Microsoft.ML.StandardLearners/Standard/LogisticRegression/MulticlassLogisticRegression.cs

```diff
@@ -69,8 +70,14 @@ public sealed class Arguments : ArgumentsBase
 
         protected override int ClassCount => _numClasses;
 
+        public MulticlassLogisticRegression(IHostEnvironment env, string featureColumn, string labelColumn,
+            string groupIdColumn = null, string weightColumn = null, Action<Arguments> advancedSettings = null)
+            : base(env, featureColumn, labelColumn, weightColumn, groupIdColumn, advancedSettings)
```

Comment:
>labelColumn [](start = 39, length = 11)

shape of the label column for multi-class is different than regression, will want to construct it here.

Reply:
I kept it as it is now and the tests seem to be passing correctly. Am I missing something?

---
In reply to: [219326153](https://github.com/dotnet/machinelearning/pull/957#discussion_r219326153) [](ancestors = 219326153)

Reply:
i think the label for multiclass is a key, and item type U4. R4 is probably acceptable, but you need it to be a key. 

---
In reply to: [219349397](https://github.com/dotnet/machinelearning/pull/957#discussion_r219349397) [](ancestors = 219349397,219326153)